python3/tests.py

```diff
@@ -0,0 +1,102 @@
+import unittest
+import os
+import pyinotify
+import tempfile
+import shutil
+
+event_holder = []
+
+
+class processor(pyinotify.ProcessEvent):
```

Comment:
This is a copy of code introduced in https://github.com/smarkets/pyinotify-smarkets/commit/fdb18c1031eeb0789cb968bd9cbe3d0988357a25